qt/tools/genhooks_gui.py

```diff
@@ -19,6 +19,17 @@
 hooks = [
     # Reviewing
     ###################
+    Hook(
+        name="overview_refreshed",
+        args=["overview: aqt.overview.Overview"],
+        doc="""Allow to update the overview window. E.g. add the deck name in the
+        title.""",
+    ),
+    Hook(
+        name="deckBrowser_rendered",
```

Comment:
please use deck_browser_did_render instead

and overview_did_refresh

otherwise looks good, thanks!